src/response.rs

```diff
@@ -1,19 +1,63 @@
 //! Iron's HTTP Response representation and associated methods.
 
-use std::io::{self, Read, Write};
+use std::io::{self, Write};
 use std::fmt::{self, Debug};
+use std::fs::File;
 
 use typemap::TypeMap;
 use plugin::Extensible;
-use modifier::{Set, Modifier};
-
+use modifier::{Set, Modifier}; 
 use hyper::header::Headers;
 
 use status::{self, Status};
 use {Plugin, headers};
 
 pub use hyper::server::response::Response as HttpResponse;
-use hyper::net::Fresh;
+use hyper::net::{Fresh, Streaming};
```

Comment:
can you `pub use` `Streaming` so implementors don't have to depend on hyper explicitly to implement `WriteBody`.